lib/scout_apm/instruments/action_controller_rails_2.rb

```diff
@@ -6,6 +6,7 @@ class ActionControllerRails2
       def initialize(context)
         @context = context
         @installed = false
+        @agent_context = ScoutApm::Agent.instance.context
```

Comment:
context is passed in right above, in the `@context` variable. And it doesn't look like this instance variable is even used?